requirements_dev.in

```diff
@@ -18,3 +18,7 @@ pyyaml==6.0
 rcssmin==1.1.1
 webtest==3.0.0
 xmltodict==0.13.0
+types-PyYAML==6.0.12.20250822
```

Comment:
Should these match the corresponding lib versions? I would also put the types together with their libs (possibly in requirements.in if needed) so that developers remember to update both the lib and the types at the same time.